pkg/html/html.go

```diff
@@ -727,6 +731,188 @@ func testImpactingComponents(testImpactingBugs []bugsv1.Bug) string {
 	return s
 }
 
+func summaryJobsFailuresByBugzillaComponent(report, reportPrev sippyprocessingv1.TestReport, endDay int, release string) string {
+	failuresByBugzillaComponent := util.SummarizeJobsFailuresByBugzillaComponent(report)
+	failuresByBugzillaComponentPrev := util.SummarizeJobsFailuresByBugzillaComponent(reportPrev)
+
+	s := fmt.Sprintf(`
+	<table class="table">
+		<tr>
+			<th colspan=4 class="text-center"><a class="text-dark" title="Bugzilla components ranked by maximum fail percentage of any job." id="BZToWorstJobImpact" href="#BZToWorstJobImpact">Bugzilla Components to Worst Job Impact</a></th>
```

Comment:
```suggestion
			<th colspan=4 class="text-center"><a class="text-dark" title="Bugzilla components ranked by maximum fail percentage of any job." id="BZToWorstJobImpact" href="#BZToWorstJobImpact">Job Impacting BZ Components</a></th>
```

Reply:
(and elsewhere)

Reply:
```
Bugzilla components ranked by maximum fail percentage of any job
```
suggestion:

Bugzilla components ranked by computing which job they contribute the most failure percentage to and then sorting the components by each one's worst job performance.

but i wonder if this would be clearer if you just showed which job was the top failing one for the component, in the top line results, and the expansion just showed "other jobs" or "all jobs"